functorch/experimental/__init__.py

```diff
@@ -1,5 +1,6 @@
-from .batch_norm_replacement import replace_all_batch_norm_modules_
 # PyTorch forward-mode is not mature yet
-from .._src.eager_transforms import jvp, jacfwd, hessian
+from .._src.eager_transforms import hessian, jacfwd, jvp
 from .._src.vmap import chunk_vmap
+from .batch_norm_replacement import replace_all_batch_norm_modules_
+from .map import map
```

Comment:
meta-point on API: (1) we should be consistent for the control flow ops, (2) we should avoid exposing implementation details publicly (functorch.experimental.map is technically public API, we will be sad if people start relying on functorch.experimental.map.trace_map for example).

For (1) consistency: cond is functorch.experimental.cond.cond. map is functorch.experimental.map. We are likely going to add some more control flow ops in the future. Let's try to gather all the APIs into a functorch.experimental.control_flow namespace.

For (2) private APIs: the implementation of map should go into functorch.experimental._map.py or functorch/_src/map.py and then a single public API should be exposed in functorch.experimental.control_flow. We didn't do this with cond, but it is good practice.

Reply:
good point, I was wondering whether experimental sounds "private" enough. I can make the adjustments proposed here.